src/app/Plans/components/VMMigrationDetails.tsx

```diff
@@ -103,12 +102,10 @@ const VMMigrationDetails: React.FunctionComponent = () => {
 
   const migrationsQuery = useMigrationsQuery();
   const latestMigration = findLatestMigration(plan || null, migrationsQuery.data?.items || null);
-  const warmPlanState = getWarmPlanState(plan || null, latestMigration, migrationsQuery);
+  const planState = getPlanState(plan || null, latestMigration, migrationsQuery);
   const isShowingPrecopyView =
     !!plan?.spec.warm &&
-    (warmPlanState === 'Starting' ||
-      warmPlanState === 'Copying' ||
-      warmPlanState === 'AbortedCopying');
+    (planState === 'Starting' || planState === 'Copying' || planState === 'AbortedCopying');
```

Comment:
I think we need to replace `planState === 'AbortedCopying'` with `planState === 'CanceledCopying' || planState === 'FailedCopying'`